TOR/Contribution.md

```diff
@@ -0,0 +1,46 @@
+# Contributing
+
+Thank you for your interest in contributing to the ETF project.
+
+Please first discuss the improvement you wish to make via
+[ETF Improvement Proposal (EIP) issue](https://github.com/etf-validator/governance/issues/new?template=etf-improvement-proposal--eip-.md)
+in our governance repository before creating a pull request.
+Pull Requests can be created for code changes that only fix bugs without an EIP,
+but need to reference an existing bug in the
+[ETF webapp](https://github.com/etf-validator/etf-webapp/issues)
+repository.
+
+Please read and follow our
+[Code of Conduct](https://github.com/guadaltel/governance/blob/8791bedab49372132c2814b95837b4401bfaee5e/CODE_OF_CONDUCT.adoc)
+before you interact with the ETF community.
+
+Before we can merge you contribution, we must have received you signed
+[ETF Entity Contributor License Agreement](https://github.com/etf-validator/governance/blob/c776741d4b1548a7019f55e8873be5c4c1b572c1/CLA/Entity.md).
+
+The ETF project is governed by a
+[Steering Group](https://github.com/etf-validator/governance/blob/master/TOR/Steering_Group.adoc)
+which is supported by a
+[Technical Committee (TC)](https://github.com/etf-validator/governance/blob/master/TOR/Technical_Committee.adoc)
+in technical matters. Your Pull Request will be reviewed by the TC.
+
+## Pull Request Process
+
+1. Fork the repository.
+2. Create a branch in your forked repository that either references an
+   ETF Improvement Proposal (EIP) or an reported issue. The branch name must
+   be 'EIP-NUMBER' for an ETF Improvement Proposal, where NUMBER is the
+   issue number from governance repository or 'issue-NUMBER' where NUMBER is
+   the issue number from the ETF webapp repository.
+3. Make the code changes. Only make changes for one concern.
```

Comment:
Not quite clear, what "Only make changes for one concern" means. Also, this sounds like a requirement for the code change, not a step.

Reply:
Ok, good point this wording is mainly used in the SE world. I also agree that this could be in a separate list.